io.fairyproject.platforms/bukkit-platform/build.gradle

```diff
@@ -48,6 +48,10 @@ repositories {
     }
 }
 
+shadowJar {
+    relocate "net.kyori", relocate + "kyori"
```

Comment:
Relocation seems necessary but my concern is that this causes some inconsistent for example Paper's adventure cannot be used for fairy, maybe we can made adventure also runtime library and download it depend if the library is loaded by any other plugin/server software?

Reply:
The problem is that if we don't relocate and get this error:
```console
[19:28:09 ERROR]: Error occurred while enabling Fairy v0.5.1b9 (Is it up to date?)
java.lang.NoSuchFieldError: LOCALE
        at net.kyori.adventure.platform.bukkit.BukkitAudience.contributePointers(BukkitAudience.java:117) ~[?:?]
        at net.kyori.adventure.platform.facet.FacetAudience.pointers(FacetAudience.java:449) ~[?:?]
        at net.kyori.adventure.pointer.Pointered.getOrDefault(Pointered.java:65) ~[patched_1.17.1.jar:git-Airplane-48]
        at net.kyori.adventure.platform.facet.FacetAudienceProvider.addViewer(FacetAudienceProvider.java:114) ~[?:?]
        at net.kyori.adventure.platform.bukkit.BukkitAudiencesImpl.<init>(BukkitAudiencesImpl.java:100) ~[?:?]
        at net.kyori.adventure.platform.bukkit.BukkitAudiencesImpl$Builder.lambda$build$1(BukkitAudiencesImpl.java:174) ~[?:?]
        at java.util.HashMap.computeIfAbsent(HashMap.java:1224) ~[?:?]
        at java.util.Collections$SynchronizedMap.computeIfAbsent(Collections.java:2722) ~[?:?]
        at net.kyori.adventure.platform.bukkit.BukkitAudiencesImpl$Builder.build(BukkitAudiencesImpl.java:174) ~[?:?]
        at net.kyori.adventure.platform.bukkit.BukkitAudiencesImpl.instanceFor(BukkitAudiencesImpl.java:89) ~[?:?]
        at net.kyori.adventure.platform.bukkit.BukkitAudiences.create(BukkitAudiences.java:53) ~[?:?]
        at io.fairyproject.bukkit.FairyBukkitPlatform.enable(FairyBukkitPlatform.java:91) ~[?:?]
        at io.fairyproject.bukkit.plugin.FairyInternalPlugin.onEnable(FairyInternalPlugin.java:17) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.17.1.jar:git-Airplane-48]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[patched_1.17.1.jar:git-Airplane-48]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[patched_1.17.1.jar:git-Airplane-48]
        at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:518) ~[patched_1.17.1.jar:git-Airplane-48]
        at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:432) ~[patched_1.17.1.jar:git-Airplane-48]
        at net.minecraft.server.MinecraftServer.loadWorld(MinecraftServer.java:710) ~[patched_1.17.1.jar:git-Airplane-48]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:309) ~[patched_1.17.1.jar:git-Airplane-48]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1197) ~[patched_1.17.1.jar:git-Airplane-48]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[patched_1.17.1.jar:git-Airplane-48]
        at java.lang.Thread.run(Thread.java:831) [?:?]
```
This is also the developer's suggestion:
![image](https://user-images.githubusercontent.com/42841502/144758911-e1873716-fa60-426e-a26b-e6cf06ea7e3b.png)
The problem is that when I relocate the Library, the main code is doesn't change only if relocated in shadowJar. So the above error remains.
```java
    ADVENTURE_API = new Library(
            "net.kyori",
            "adventure-platform-bukkit",
            "4.0.1",
            null,
            new Relocate("net{}kyori", IMANITY_LIB_PACKAGE + "kyori")
    );
```

Reply:
make sense, will then keep it shade for now is fine I guess
I will make a PR target to reduce shaded library soon